wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,29 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
+- **Use the [discussion page](/wiki/Beatmap_discussion).** Discuss your suggestions with other community members. Otherwise, it is unlikely your concerns will be heard by anyone with the power to address them appropriately.
 
 ## Responding to a mod
 
-Someone just posted some suggestions on your map, great! However, it doesn't simply end there. For the record, it's necessary to give a proper response for two major reasons. Firstly, so that future mods can be tailored more specifically to your concept or style, resulting in fewer unhelpful suggestions. Secondly, just as a mod post *helps you improve your mapping ability*, providing a detailed mod response helps the modder improve their modding ability, meaning they gain experience and can provide more helpful and detailed mods in the future. You never know who you may need to ask for a mod next time around!
-
-- Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them — even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap
-- As with making a mod post, be sure to explain yourself as much as possible. State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead.
-- When declining a suggestion, remain calm and professional, say what needs to be said in order to explain yourself and **leave it at that**. No matter how impolite, confusing or unhelpful the suggestion was, getting angry and responding with hostility, sarcasm, etc. will not help the situation.
-- [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) members will review previous mods before placing any icons and may want to initiate or revisit discussions during their checks. Any assistance you can provide as to why you did or didn't do something takes very little effort, and could be a huge help to all parties later on.
+- **Treat all suggestions as valid and equally important.** Don't draw hasty conclusions about the validity of a suggestion without properly assessing it.
+- **Explain yourself as much as possible.** State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead. This will not only help the modder to learn, but also allow future mods to be tailored to your concept or style.
+- **Remain calm.** Say what needs to be said in order to explain yourself and leave it at that. No matter how unhelpful the suggestion was, responding with hostility, sarcasm, etc. will not help the situation.
+- **Provide context.** By providing concise explanations on the mods you responded to, you make it easier for future modders, [Beatmap Nominators](/wiki/People/Beatmap_Nominators), and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) to revisit discussions during their checks.
```

Comment:
I hope this doesn't come as nitpicky. 

Considering that you have abbreviated Beatmap Nominator and Nomination Assessment Team in line 28 wouldn't it be good to add the abbreviations here as the original article has it? Just for the sake of "this has been seen so we know what it means" type of reading.

```suggestion
- **Provide context.** By providing concise explanations on the mods you responded to, you make it easier for future modders, [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN), and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) to revisit discussions during their checks.
```

Reply:
Definitely would be suitable to add the abbreviations in as that is what the modders see on the discussion page on NATs and BNs